assignments/lambda-classes.js

```diff
@@ -1 +1,180 @@
 // CODE here for your Lambda Classes
+class Person {
+    constructor(personAttributes) {
+        this.name = personAttributes.name;
+        this.age = personAttributes.age;
+        this.location = personAttributes.location;
+    }
+    speak() {
+        console.log(`Hello my name is ${this.name}, I am from ${this.location}.`);
+    }
+}
+
+class Instructor extends Person {
+    constructor(instructorAttributes) {
+        super(instructorAttributes);
+        this.specialty = instructorAttributes.specialty;
+        this.favLanguage = instructorAttributes.favLanguage;
+        this.catchPhrase = instructorAttributes.catchPhrase;        
+    }
+    demo(subject) {
+        console.log(`Today we are learning about ${subject}`);
+    }
+    grade (student, subject) {
+        console.log(`${student.name} receives a perfect score on ${subject}`);
+    }
+    appointGrade(student) {
+        student.grade = Math.round(Math.random() * 100);
+        console.log(`${student.name}'s grade is now ${student.grade}`);
+    }
+}
+
+class Student extends Person {
+    constructor(studentAttributes) {
+        super(studentAttributes);
+        this.previousBackground = studentAttributes.previousBackground;
+        this.className = studentAttributes.className;
+        this.favSubjects = studentAttributes.favSubjects;
+        this.grade = studentAttributes.grade;
+    }
+    listsSubjects(favSubjects, length) {
+        for (let i = 0; i < length; i++) {
+            console.log(favSubjects[i]);
+        }
+    }
+    PRAssignment(subject) {
+        console.log(`${this.name} has submitted a PR for ${subject}`);
+    }
+    sprintChallenge(subject) {
+        console.log(`${this.name} has begun sprint challenge on ${subject}`);
+    }
+    graduate() {
+        if (this.grade === 0) {
+            console.log(`${this.name} has completely failed at Lambda school!!!`)
+        }
+        else {
```

Comment:
Best practice syntax is -
graduate() {
        if (this.grade === 0) {
            console.log(`${this.name} has completely failed at Lambda school!!!`)
        } else if (this.grade > 70) {
                console.log(`${this.name} has graduated from Lambda school!!!`)
            } else {
                console.log(`${this.name} is not ready to graduate.`)
            }
        }

Yours works fine, readability issue